lighthouse-core/test/audits/uses-rel-preconnect-test.js

```diff
@@ -250,4 +250,76 @@ describe('Performance: uses-rel-preconnect audit', () => {
       {url: 'http://cdn.example.com', wastedMs: 150},
     ]);
   });
+
+  it(`should return code 1 if preconnect links equal length 6`, async () => {
```

Comment:
drop this test and just assert warnings.length === 0  in the previous test (like at L252)

Reply:
done